Filters/HueParameters.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PhotoEnhancer
+{
+    public class HueParameters : IParameters
+    {
+        public double Coefficient { get; set; }
+        public ParameterInfo[] GetDecription()
+        {
+            return new[]
+            {
+                new ParameterInfo()
+                {
+                    Name = "Коэффициент",
```

Comment:
Название "Коэффициент" не отражает смысл параметра и пользователю будет непонятно.
```suggestion
                    Name = "Сдвиг оттенка (в °)",
```